yt_dlp/extractor/redbee.py

```diff
@@ -280,69 +292,79 @@ def _real_extract(self, url):
         if provider in self._PROVIDERS:
             return self.url_result(data['url'], self._PROVIDERS[provider])
 
-        title = data['subtitle']
+        # In general, we prefer the 'subtitle' because it's a lot more specific than the 'title',
+        # but we need to fallback to 'title' in case 'subtitle' doesn't exist, like for livestreams.
+        # https://github.com/yt-dlp/yt-dlp/issues/4656
+        title = data.get('subtitle') or data['title']
         is_live = data.get('isLive')
         height_re = r'-(\d+)p\.'
-        formats = []
+        formats, subtitles = [], {}
 
         m3u8_url = data.get('urlHlsAes128') or data.get('urlHls')
-        if m3u8_url:
-            formats.extend(self._extract_m3u8_formats(
-                m3u8_url, media_id, 'mp4', m3u8_id='hls', fatal=False))
-
-        fix_url = lambda x: x.replace('//rtbf-vod.', '//rtbf.') if '/geo/drm/' in x else x
-        http_url = data.get('url')
-        if formats and http_url and re.search(height_re, http_url):
-            http_url = fix_url(http_url)
-            for m3u8_f in formats[:]:
-                height = m3u8_f.get('height')
-                if not height:
-                    continue
-                f = m3u8_f.copy()
-                del f['protocol']
-                f.update({
-                    'format_id': m3u8_f['format_id'].replace('hls-', 'http-'),
-                    'url': re.sub(height_re, '-%dp.' % height, http_url),
-                })
-                formats.append(f)
-        else:
-            sources = data.get('sources') or {}
-            for key, format_id in self._QUALITIES:
-                format_url = sources.get(key)
-                if not format_url:
-                    continue
-                height = int_or_none(self._search_regex(
-                    height_re, format_url, 'height', default=None))
+        # The old api still returns m3u8 and mpd manifest for livestreams, but these are 'fake'
+        # since all they contain is a 20s video that is completely unrelated.
+        # https://github.com/yt-dlp/yt-dlp/issues/4656#issuecomment-1214461092
```

Comment:
Do
```py
m3u8_url = None if data.get('isLive') else traverse_obj(data, 'urlHlsAes128', 'urlHls')
```

This allows you to de-indent this whole large block to it's previous state